libraries/support/azure/service.rb

```diff
@@ -14,6 +14,11 @@ def fetch(key)
       end
     end
 
+    EXEMPTED_TAGS = %i(
```

Comment:
`EXEMPTED_ATTRIBUTES` / `EXEMPTED_FIELDS`? In case we find other hashes we want to keep, which arent called `tags`?